src/main/java/io/kuzzle/sdk/core/Kuzzle.java

```diff
@@ -246,12 +249,13 @@ public Kuzzle(@NonNull final String url, final KuzzleOptions options, final Kuzz
     this.defaultIndex = opt.getDefaultIndex();
     this.headers = opt.getHeaders();
     this.metadata = opt.getMetadata();
+    this.port = opt.getPort();
     this.queueMaxSize = opt.getQueueMaxSize();
     this.queueTTL = opt.getQueueTTL();
     this.reconnectionDelay = opt.getReconnectionDelay();
     this.replayInterval = opt.getReplayInterval();
 
-    this.url = url;
+    this.url = "http://" + host + ":" + this.port;
```

Comment:
Actually, adding SSL support involves more work than just using `https` or `wss` in the URL: we also have to manage certificates, either to ignore them and/or (preferably "and") by allowing clients to use one.
I think that adding SSL support should be done in a separate PR.